P5/Source/Specs/additions.xml

```diff
@@ -14,8 +14,7 @@ $Id$
 within a manuscript or other object, such as marginalia or other annotations.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">방주 또는 기타 주석과 같이 원고 내의 중요한 부가정보의 기술을 포함한다.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含手稿中任何重要增添文字的描述，例如旁註或其他註釋。</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料中にある，重要な付加物を示す．例えば，欄外の書き込みや
-  注釈など．</desc>
+  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料中にある，重要な付加物を示す．例えば，欄外の書き込みや注釈など．</desc>
```

Comment:
`@versionDate` should be updated

Reply:
Dear Peter, thank you for your cheking. I've updated the date.